frontend/demo/src/assets/images/icons/steps-2.svg

```diff
@@ -0,0 +1 @@
675-.2589084.85250338-.4627875c.2453766-.2038791.4402346-.4555706.5845738-.7550745.1443391-.2995038.2165087-.64411362.2165087-1.03382945 0-.45827695-.1154713-.83897159-.346414-1.1420839s-.5809653-.52864231-1.05006769-.67658999c.19485792-.06495263.36896708-.14614343.52232747-.24357239.15336042-.09742896.28326572-.2165088.38971582-.35723951.1064502-.14073072.187641-.30581868.2435724-.49526387.0559314-.1894452.0838972-.40866036.0838972-.65764547 0-.28506991-.0550293-.55029319-.165088-.79566982-.1100586-.24537664-.2679296-.45917908-.47361298-.64140731-.20568335-.18222824-.45376635-.3247632-.74424898-.42760487-.29048264-.10284168-.61614795-.15426252-.97699594-.15426252-.34641408 0-.67027515.04781236-.97158322.14343708-.30130808.09562472-.56833559.23815967-.80108255.42760487-.23274695.18944519-.42670275.42580063-.58186739.7090663-.15516463.28326568-.25981055.61253947-.31393775.98782138l.70365359.12449256c.06495263.01082544.12629679.01623816.18403247.01623816.12268832 0 .22192152-.02976996.2976996-.08930988.07577807-.05953992.13531799-.15787099.17861975-.29499323.08299504-.26341904.22913848-.47902571.43843031-.64682003.20929184-.16779431.47271087-.25169147.79025711-.25169147.35002255 0 .62607127.0947226.82814614.28416779.20207488.1894452.30311232.47000451.30311232.84167794 0 .17320704-.02525936.33198016-.07577808.47631936-.05051872.14433919-.1398286.26792963-.26792964.37077131s-.30040595.18313035-.51691475.24086603c-.21650879.05773568-.48894903.08840776-.8173207.09201624v.9526387c.39693279 0 .7135769.03067208.94993234.09201624.23635543.06134416.41768155.14614344.54397835.25439784.1262968.10825439.20929183.23906179.24898511.39242219.03969328.15336039.05953992.32386107.05953992.51150203 0 .14794767-.02796572.29499323-.08389716.44113669-.05593143.1461434-.14073071.2787551-.25439783.3978349s-.25529996.2156067-.42489851.2895805c-.16959856.0739738-.36806495.1109608-.59539919.1109608-.20207488 0-.37618403-.02977-.52232747-.0893099s-.27334235-.1371223-.38159675-.232747-.20117276-.2047812-.27875508-.3274695-.14884979-.2471809-.21380243-.3734777c-.03969328-.07577808-.09652684-.13351375-.17050068-.17320703s-.157871-.05953992-.25169147-.05953992c-.10464592 0-.2074876.02165088-.30852504.06495264l-.58998647.24357239c.1082544.33558862.23815968.63148402.38971584.88768602.15155615.2562021.33558863.4727109.55209743.6495264.21650879.1768155.47090663.3103293.7631935.4005413s.63509247.135318 1.02841678.135318z"/></g></g><path d="m8 16c-4.418278 0-8-3.581722-8-8s3.581722-8 8-8 8 3.581722 8 8-3.581722 8-8 8zm0-1c3.8659932 0 7-3.1340068 7-7 0-3.86599325-3.1340068-7-7-7-3.86599325 0-7 3.13400675-7 7 0 3.8659932 3.13400675 7 7 7zm2.5171573-9.28284271c.1562097-.15620972.4094757-.15620972.5656854 0 .1562097.15620971.1562097.40947571 0 .56568542l-3.99999999 3.99999999c-.15620971.1562097-.40947571.1562097-.56568542 0l-1.6-1.59999999c-.15620972-.15620971-.15620972-.40947571 0-.56568542.15620971-.15620972.40947571-.15620972.56568542 0l1.31715729 1.31715729z" fill="#29a979"/></g></svg>
```

Comment:
Lots of assets that we do not need right now. E.g. do we use this one? :)

Reply:
I just moved all of them, but I will clean up a bit